src/stylesheets/app.scss

```diff
@@ -1 +1,5 @@
 @import './ticket_sidebar.scss';
+
+body {
+  margin: 0;
```

Comment:
this is to remove the default margin present in the browser stylesheet and fit in the ticket sidebar. we may want to make this conditional, ie. only for sidebar apps, but I was hoping that developers could just delete/change this if that's needed.